salt/utils/schedule.py

```diff
@@ -1320,10 +1327,15 @@ def _handle_until(job, data):
                     del data[item]
             run = False
 
+            if 'name' in data:
+                job_name = data['name']
+            else:
+                job_name = data['name'] = job
```

Comment:
The call to `_handle_until` was updated to not pass `job` so wouldn't this fail?

Reply:
Never mind.  Makes more sense when looking at the full code.